src/components/SavedFiltersMenu.vue

```diff
@@ -32,11 +42,13 @@
   import SaveFilterModal from '@/components/SaveFilterModal.vue'
   import { useShowModal } from '@/compositions'
   import { useCan } from '@/compositions/useCan'
+  import { useCustomDefaultFlowRunsFilter } from '@/compositions/useCustomDefaultFlowRunsFilter'
+  import { SavedFlowRunsSearch } from '@/compositions/useSavedFlowRunsSearches'
   import { SavedSearch } from '@/models/SavedSearch'
-  import { customSavedSearch } from '@/utilities/savedFilters'
+  import { customSavedSearch, systemDefaultSavedSearch } from '@/utilities/savedFilters'
 
   const props = defineProps<{
-    savedSearch: SavedSearch | null,
+    savedSearch: SavedFlowRunsSearch | null,
```

Comment:
This type change just adds an `isDefault` property onto the object. trying to keep `SavedSearch` as a one-to-one api/backend entity and then this composition wraps that to include the (atm) FE-only user-customizable default and to also be more context-aware as saved _flow runs_ search.